test-case-reporting/static/build-list.html

```diff
@@ -0,0 +1,17 @@
+<html ⚡>
+<head>
+  <title>Builds</title>
+</head>
+<body>
+  <ul>
+  {{#builds}}
+  <li><a href="/test-results/build/{{buildNumber}}">Build {{buildNumber}}</a></li>
```

Comment:
nit: intendation, possibly break across multiple lines if past 80 characters. Also, not sure if you did this elsewhere, but since the page isn't an AMP page it shouldn't have the lightning bolt in the html tag. just want a `<!DOCTYPE html>` maybe

Reply:
Done.